src/main/java/stsjorbsmod/JorbsMod.java

```diff
@@ -440,6 +439,9 @@ public void receiveStartAct() {
             }
             ExertedField.exerted.set(c, false);
         }
+        SkillsPlayedThisActSaveData sp =
+                (SkillsPlayedThisActSaveData)BaseMod.getSaveFields().get(MOD_ID + ":SkillsPlayedThisAct");
+        sp.skillsPlayed = 0;
```

Comment:
This is just immediately discarding whatever `BaseMod.getSaveFields().get` is loading, no? Can this not juts be `SkillsPlayedThisActSaveData.skillsPlayed = 0;`?